disko-install

```diff
@@ -19,6 +19,10 @@ Usage: $0 [OPTIONS]
   --write-efi-boot-entries    Write EFI boot entries to the NVRAM of the system for the installed system.
                               Specify this option if you plan to boot from this disk on the current machine,
                               but not if you plan to move the disk to another machine.
+  --system-option KEY VALUE   Pass the specified system option to the NixOS configuration.
+                              This option can be specified multiple times.
+                              For example, to set the hostname, use
+                              --system-option users.users.root.openssh.authorizedKeys.keys '[ "ssh-rsa ..." ];'
```

Comment:
Is there a reason to require the semicolons here? Why don't we either add them in the serialization step below or to just let user pass a full nix attrset as a string here?

Reply:
@phaer Hmm you are right I should add the semicolon in the serialization step. I want to make the option human readable so that's why I don't want a full attrset.